### DISCUSSION TITLE
Mirth Connect 4.5.0 - Error converting CSV to PDF

### DISCUSSION THREAD

**Discussion by davidberg-ot** [nextgenhealthcare/connect#6280] [Q&A]
Excuse me if I'm not using the proper terminology, but I have a user who can put a properly formatted CSV into the IN folder for a channel and it's supposed to convert to a PDF and save in the OUTBOX folder.  We are getting an error in the server log

ERROR SERVER:144
[FATAL ERROR] 1:10: DOCTYPE is disallowed when the feature "http://apache.org/xml/features/disallow-doctype-decl" set to true.

I have seen some posts in the older forum but cannot figure out which file needs to be udpated or even if that is the appropriate fix.  Any assistance would be greatly appreciated!

**Comment by jonbartels** [comment]
> I have seen some posts in the older forum but cannot figure out which file needs to be udpated or even if that is the appropriate fix.

Please link to the posts you found already. That can get help faster because we can see what you already tried or clear up a more specific confusion.

When do you get this error? From a source transformer or destination transformer? Can you show more of the error stack trace so we know where the error is happening in Mirth?

**Reply by davidberg-ot** [reply]
@jonbartels 
It's not clear from this comment https://github.com/nextgenhealthcare/connect/discussions/5449 what file to edit.  And it's also referring to 4.1.1 not 4.5.0, so not sure if it still applies.

**Reply by pacmano1** [reply]
You are going to need to post a lot more information.    I suspect this is coming from the pdf template in the document writer step.   @davidberg-ot, while this is not the mirth forums, please read https://forums.mirthproject.io/articles/177082-forum-etiquette, it is a helpful guide on how to ask for help.
